roles/edpm_update/tasks/main.yml

```diff
@@ -19,12 +19,21 @@
 
 - name: Apply kernel patch via kpatch
   ansible.builtin.include_tasks: kpatch.yml
-  when: edpm_update_enable_kpatch
+  when:
+  - edpm_update_enable_kpatch
+  - not ansible_local.bootc
 
 - name: Update packages
   ansible.builtin.include_tasks: packages.yml
-  when: edpm_update_enable_packages_update
+  when:
+  - edpm_update_enable_packages_update
+  - not ansible_local.bootc
 
 - name: Update containers
   ansible.builtin.include_tasks: containers.yml
-  when: edpm_update_enable_containers_update
+  when:
+  - edpm_update_enable_containers_update
+
+- name: Update OS (bootc)
+  ansible.builtin.include_tasks: bootc.yml
+  when: ansible_local.bootc
```

Comment:
Is there a particular reason this comes after the containers update? My instinct would be to update the system first (the kernel and other system-tied components can provide new interfaces/features that the new containers can make use of, not vice versa).

Reply:
Actually i see the bootc update task file does not trigger a reboot so the order here doesn't matter.

Reply:
Yeah, that's right. This step just writes that new ostree layer and then we wait until the user is ready to reboot.

Reply:
It's not consistent with how package mode is done though. In that, there will be a period of time when new containers are running with old OS. If the user forgets to reboot or it's skipped, that could be an unsupported state.

We might need to do the bootc update first, force a reboot and verify it was done, then update containers.

Reply:
Move it now, but I'm reluctant to force a reboot.